packages/design/src/Form/components/DateOfBirth/DateOfBirth.tsx

```diff
@@ -66,37 +80,40 @@ export const DateOfBirthPattern: PatternComponent<DateOfBirthProps> = ({
             Day
           </label>
           <input
-            className="usa-input"
-            aria-describedby="mdHint"
+            className={classNames('usa-input', {
+              'usa-input--error': error,
+            })}
             id={dayId}
             {...register(dayId)}
             minLength={2}
             maxLength={2}
             pattern="[0-9]*"
             inputMode="numeric"
+            aria-describedby={
```

Comment:
This is described by `hint-${monthId}` as well. You'll have identical `aria-describedby` logic for all 3 form fields, so you many want to make a function for it.

Reply:
Good idea! Updated!